content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -87,6 +87,7 @@ different Kubernetes components.
 | `CSRDuration` | `true` | Beta | 1.22 | |
 | `ConfigurableFSGroupPolicy` | `false` | Alpha | 1.18 | 1.19 |
 | `ConfigurableFSGroupPolicy` | `true` | Beta | 1.20 | |
+| `ConfigurableFSGroupPolicy` | `true` | GA | 1.23 | |
```

Comment:
Hi. I'm afraid this needs to move out of the section “Feature gates for Alpha or Beta features” and under the heading for graduated features.

Reply:
fixed.

Reply:
confirming this requested change landed.